### ISSUE TITLE
NV 375 point

### ISSUE THREAD

**Issue by jteresco** [TravelMapping/HighwayData#124]
I'd like to add a point so I can get credit for the short piece of NV 375 I drove today.  I turned around at the "Alien Research Center" probably less than a mile from NV 318.  "AliResCen" seem like an OK label?

**Comment by ovoss** [comment]
I can see the location of the ARC on Google Maps, but not in OSM/Mapnik. 
However, MQOpenSat open-source satellite imagery shows something that looks 
like the ARC, and an access road identified in Bing Map as Cactus Ridge 
Lane. In keeping with our normal use of intersections for waypoints, I 
suggest going with:

CacRidLn http://www.openstreetmap.org/?lat=37.532212&lon=-115.241829

BTW, anything interesting at the ARC? When I clinched NV 375 in 1998, the 
town of Rachel (far from NV 318, but close to Area 51) was the UFO/Area 51 
buff hangout, and there wasn't anything near the south end of NV 375 except 
an Extraterrestrial Highway sign (not there on a side trip during a later 
trek up US 93).

----- Original Message ----- 
From: Jim Teresco
To: TravelMapping/HighwayData
Sent: Sunday, October 04, 2015 11:26 PM
Subject: [HighwayData] NV 375 point (#124)


I'd like to add a point so I can get credit for the short piece of NV 375 I 
drove today. I turned around at the "Alien Research Center" probably less 
than a mile from NV 318. "AliResCen" seem like an OK label?
—
Reply to this email directly or view it on GitHub.



--
Oscar Voss - oscar.voss@comcast.net - Arlington VA

my Hot Springs and Highways pages (NEW ADDRESS):
http://www.alaskaroads.com/home.html
